web-console/developer/add/sections/pipelines/pipelines.md

```diff
@@ -0,0 +1,28 @@
+# Pipelines
+The Pipelines section allows the user to enable a pipeline template for their application, component or service.
+
+## Default state
+A Pipelines section appears in +Add flows (for Import from Git,  Container Image, and From Dockerfile) when the Pipelines operator is installed.
+* When the Pipelines operator is not installed, this section is not displayed
+* The Pipelines section has a “Dev Preview” badge
+* Under this Pipelines section, there is a checkbox control labeled “Add pipeline”
+* The default state of the Add pipeline checkbox is OFF
+
+![Add pipeline](img/+Add_Flow_Pipelines_default.png)
+
+## Checked state
+When the “Add pipeline” checkbox is checked a "Show pipeline visualization" expandable is displayed.
+* The Expandable control is contracted by default
+
+![Show visualization](img/+Add_Flow_Pipelines_vizclosed.png)
+## Expanded state
+When the “Show pipeline visualization” control is expanded, a pipeline visualization is displayed below the expandable control.
+* The expandable's text changes to “Hide pipeline visualization”
+
+![Show visualization open](img/+Add_Flow_Pipelines_vizopen.png)
+
+## Alert
+When no pipeline template is available, show an inline alert
+Alert should say “Since there are no pipeline templates available for this runtime, a pipeline cannot be created.”
```

Comment:
@beaumorley can you remove the "Alert should say..." piece? That way if it changes in the future, we can just update the mock.  Also please not the severity level of the inline alert!

Reply:
OK. Updated based on feedback.